web/ce/store/timeline/base-timeline.store.ts

```diff
@@ -44,7 +44,11 @@ export interface IBaseTimelineStore {
   updateActiveBlockId: (blockId: string | null) => void;
   updateRenderView: (data: any) => void;
   updateAllBlocksOnChartChangeWhileDragging: (addedWidth: number) => void;
-  getUpdatedPositionAfterDrag: (id: string, ignoreDependencies?: boolean) => IBlockUpdateDependencyData[];
+  getUpdatedPositionAfterDrag: (
+    id: string,
+    shouldUpdateHalfBlock: boolean,
```

Comment:
Do we need this flag?

Reply:
yeah, to make sure we only update the block's missing date, when the missing side is dragged. Either i pass down the side that is being dragged or the flag.